tern/__main__.py

```diff
@@ -81,10 +81,13 @@ def do_main(args):
         if args.name == 'lock':
             drun.execute_dockerfile(args, True)
         elif args.name == 'report':
-            if args.layer or args.layer_inclusive:
-                raise NotImplementedError("This feature is not implemented yet!")
             if args.dockerfile:
-                drun.execute_dockerfile(args)
+                if (not args.load_until_layer):
+                    drun.execute_dockerfile(args)
+                else:
+                    logger.critical("Currently layer can only be used with"
```

Comment:
small nit: -- update to say: `currently --layer/-y option can only be used with`

Reply:
done, the rest of flag mentions I put in strings got repalced to the `--long/-short` form